tpp-app/app.json

```diff
@@ -3,6 +3,7 @@
     "name": "tpp-app",
     "slug": "tpp-app",
     "version": "1.0.0",
+    "orientation": "portrait",
```

Comment:
Technically we don't need this if we are already locking it for ios and android separately but it doesn't hurt.